addons/website/static/src/css/website.sass

```diff
@@ -482,6 +482,89 @@ div.media_iframe_video
         display: none
         z-index: 2
 
+.countdown-text
```

Comment:
Please find a way to make the css smaller. Dont use css limit yourself to boostrap only classes.